model-optimizer/mo/main.py

```diff
@@ -147,8 +146,11 @@ def prepare_ir(argv: argparse.Namespace):
                     "bat" if sys.platform == "windows" else "sh"))
             # If the IE was not found, it will not print the MO version, so we have to print it manually
             print("{}: \t{}".format("Model Optimizer version", get_version()))
+            argv.ie_is_available = False
+        else:
+            argv.ie_is_available = True
```

Comment:
Resulting xml will have
`		<cli_parameters>`
`....`
`			<ie_is_available value="True"/>`
`....`

This field could be deleted in the beginning of 'emit_ir'
    if 'ie_is_available' in argv:
        del argv.ie_is_available